languages/csharp/Projects.cs

```diff
@@ -0,0 +1,74 @@
+using Newtonsoft.Json;
+
+namespace Bit.Sdk;
+
+
+public class Projects {
```

Comment:
We should name this `ProjectsClient`.

Reply:
Do you want in BitwardenClient to be called Projects or ProjectsClient?

Reply:
Done